Source/Core/Core/PowerPC/Jit64Common/EmuCodeBlock.h

```diff
@@ -108,6 +105,9 @@ class EmuCodeBlock : public Gen::X64CodeBlock
   void Clear();
 
 protected:
-  std::unordered_map<u8*, TrampolineInfo> backPatchInfo;
-  std::unordered_map<u8*, u8*> exceptionHandlerAtLoc;
+  FarCodeCache m_far_code;
+  u8* m_near_code;  // Backed up when we switch to far code.
```

Comment:
Is there any reason we would need to access these 4 variables from somewhere other than the class or a subclass?